src/libPMacc/include/particles/operations/ConcatListOfFrames.hpp

```diff
@@ -59,12 +58,24 @@ struct ConcatListOfFrames
      * @param destFrame single frame were all particles are copied in
      * @param srcBox particle box were particles are read from
      * @param particleFilter filter to select particles
-     * @param localOffset Offset of the local domain, used to calculate globalCellIdx,
-     *                    can be negative for the first GPU: localDomain.offset - globalWindow.offset
+     * @param domainOffset offset to a user-defined domain. Can, e.g. be used to
+     *                     calculate a totalCellIdx relative to
```

Comment:
maybe better /*... calculate a totalCellIdx out of globalDomain.offset + localDomain.offset*/